src/libraries/System.Security.Cryptography.Cng/tests/SymmetricCngTestHelpers.cs

```diff
@@ -319,9 +320,26 @@ public static class SymmetricCngTestHelpers
             {
                 using (SymmetricAlgorithm alg = persistedFunc(keyName))
                 {
+                    alg.Mode = CipherMode.CFB;
```

Comment:
The positive "it works with CFB8" tests were added in the CFB one-shot PR. #55480